File: RandomTesterObjC/RandomTesterObjC/MyClass.m
//
//  MyClass.m
//  RandomTesterObjC
//
//  Created by Blake Oistad on 1/5/16.
//  Copyright © 2016 Blake Oistad. All rights reserved.
//

#import "MyClass.h"

@implementation MyClass

+ (id)getStartTime {
    NSDateFormatter *formatter = [[NSDateFormatter alloc] init];
    [formatter setDateStyle:NSDateFormatterNoStyle];
    [formatter setTimeStyle:NSDateFormatterMediumStyle];
    NSString *startTime = [formatter stringFromDate:[NSDate date]];
    return startTime;
}

@end

File: RandomTesterSwift/RandomTesterSwift/MyClass.swift
//
//  MyClass.swift
//  RandomTesterSwift
//
//  Created by Blake Oistad on 1/5/16.
//  Copyright © 2016 Blake Oistad. All rights reserved.
//

import UIKit

class MyClass: NSObject {

    class func getStartTime() -> String {
        let formatter = NSDateFormatter()
        formatter.dateStyle = .NoStyle
        formatter.timeStyle = .MediumStyle
        let currentDate = NSDate()
        let startTime = formatter.stringFromDate(currentDate)
        return startTime
    }
    
}
